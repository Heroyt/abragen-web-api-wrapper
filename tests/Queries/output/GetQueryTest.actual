[
	'class' => 'issuedinvoices',
	'fulltext' => 'FVT-1/2020',
	'select' => [
		'id',
		[
			'name' => 'rows',
			'value' => [
				'field' => 'rows',
				'query' => [
					'select' => [['name' => 'storecardName', 'value' => 'storecard_id.name']],
				],
			],
		],
		[
			'name' => 'firm_id',
			'value' => ['field' => 'firm_id', 'query' => ['select' => ['firm_name', 'name']]],
		],
	],
]
