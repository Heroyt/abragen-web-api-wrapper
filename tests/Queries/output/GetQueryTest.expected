[
	'class' => 'issuedinvoices',
	'fulltext' => 'FVT-1/2020',
	'select' => [
		'id',
		[
			'name' => 'rows',
			'value' => [
				'field' => 'rows',
				'query' => [
					'select' => [['name' => 'storecardName', 'value' => 'storecard_id.name']],
				],
			],
		],
		[
			'name' => 'firm_id',
			'value' => [
				'field' => 'firm_id',
				'query' => ['select' => [['name' => 'firm_name', 'value' => 'name']]],
			],
		],
	],
]
